site/styles/_poster.scss

```diff
@@ -41,4 +41,12 @@
   max-width: 100%;
   position: relative;
   width: 100%;
+
+  &-portrait {
+    aspect-ratio: var(--image-ratio-poster-portrait);
```

Comment:
`aspect-ratio` is not supported on Safari for iOS <15... 1.92% global usage... do we care?